desktop_version/src/Script.cpp

```diff
@@ -954,8 +950,23 @@ void scriptclass::run( KeyPoll& key, Graphics& dwgfx, Game& game, mapclass& map,
 					i=obj.getcrewman(1);
 				}
 
-				obj.entities[i].rule =7;
-				obj.entities[i].tile = 6;
+				if (words[1] == "player")
+				{
+					game.gravitycontrol = !game.gravitycontrol;
+				}
+				else
+				{
```

Comment:
For code readability the `getcrewman` if/else from above could be moved  into this scope.

Reply:
Fixed in 7173fce493453e903fd01133e58c950ef5128d52.